features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapReceiverSnmp4jImpl.java

```diff
@@ -150,19 +182,203 @@ public void stop() throws InterruptedException {
     /**
      * The execution context.
      */
-    // TODO: HZN-609: Change this to TrapdNotificationListener.receive()
     @Override
     public void run() {
```

Comment:
Because the run() method is replaced by the trapReceived() method, you should just delete the content of the run() method instead of filling it with the socket code from Syslogd.